records_mover/records/airbyte/airbyte.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from http import HTTPStatus
+import requests
+from records_mover import Session
+
+logger = logging.getLogger(__name__)
+
+
+class AirbyteEngine:
+    """
+    Intention: Main engine of activity for connecting to and managing airflow "stuff."
+    General thoughts, maybe this should encapsulate making a request to airbyte
+        Thus we'd have a method for making a request which'd know how to authenticate
+    """
+
+    def __init__(self, session: Session = None):
+        """
+        Args:
+            session: Optional records mover Session, exposed for testing.
+                     If a session isn't provided, one will be requested
+        """
+        if session is None:
+            self.session = Session()
+        else:
+            self.session = session
+
+    def healthcheck(self) -> bool:
+        self.session.set_stream_logging()
+        data = self.session.creds.airbyte("local-airbyte")
```

Comment:
This can't be hard-coded to local-airbyte

Reply:
what should it be? required argument on the CLI? i'm okay with it defaulting to this maybe

Reply:
Required argument doesn't make a ton of sense so having just a single stable entry we define is enticing. In production, we'll probably only have a single stable hostname for the airbyte instance. We'll have a different instance to run integration tests against and that's about it.